src/middlewared/middlewared/plugins/ftp_/status.py

```diff
@@ -0,0 +1,25 @@
+from middlewared.service import Service, private
+from middlewared.schema import Int, returns
+from middlewared.utils.network_.procfs import read_proc_net
+
+
+class FTPService(Service):
+
+    @private
+    @returns(Int('number_of_connections'))
+    def connection_count(self):
+        ''' Return the number of active connections '''
+        # FTP listening port is 21
+        ftp = 21
+
+        try:
+            proc_data = self.middleware.call(read_proc_net)
```

Comment:
This won't work. You can just call read_proc_net() since this method is synchronous.